cmd/soroban-cli/src/commands/network/start.rs

```diff
@@ -0,0 +1,151 @@
+use std::process::{Command, Stdio};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {}
+
+const FROM_PORT: i32 = 8000;
+const TO_PORT: i32 = 8000;
+const CONTAINER_NAME: &str = "stellar";
+const DOCKER_IMAGE: &str = "stellar/quickstart";
+
+/// This command allows for starting a stellar quickstart container. To run it, you can use the following command:
+/// `soroban network start <NETWORK> [OPTIONS] -- [DOCKER_RUN_ARGS]`
+///
+/// OPTIONS: refer to the options that are available to the quickstart image:
+/// --enable-soroban-rpc - is enabled by default
+/// --protocol-version (only for local network)
+/// --limits (only for local network)
+
+/// `DOCKER_RUN_ARGS`: These are arguments to be passed to the `docker run` command itself, and should be passed in after the slop `--`. Some common options are:
```

Comment:
using the slop arg here to allow users to pass in options to `docker run`

i had considered using the slop arg for stellar/quickstart specific arguments instead, but it seemed to make more sense to use it for `docker run` options since the stellar/quickstart image has way fewer options than `docker run`